src/dbnode/integration/bootstrap_helpers.go

```diff
@@ -31,7 +33,6 @@ import (
 	"github.com/m3db/m3/src/dbnode/storage/bootstrap/bootstrapper"
 	"github.com/m3db/m3/src/dbnode/storage/bootstrap/result"
 	"github.com/m3db/m3/src/x/context"
-
```

Comment:
Hmm, there should be an empty line between the local imports and 3rd party ones. I wonder why the linter is not complaining.